zerver/actions/create_user.py

```diff
@@ -487,6 +487,7 @@ def notify_created_bot(user_profile: UserProfile) -> None:
     send_event_on_commit(user_profile.realm, event, bot_owner_user_ids(user_profile))
 
 
+@transaction.atomic(durable=True)
```

Comment:
This is not a safe change;  `send_initial_realm_messages`, for example, fails to use `savepoint=False` and thus is not safe to run inside a larger transaction. Maybe just requires changing the `transaction.atomic` in that function to pass `savepoint=False`; I didn't see anything else inside `process_new_human_user` that wasn't safe to run in a larger transaction.